### DISCUSSION TITLE
增加隐藏搜索栏推荐功能

### DISCUSSION THREAD

**Discussion by amokiel** [the1812/Bilibili-Evolved#4375] [功能建议 / Ideas]
顶栏的搜索框经常推荐一些关键词，很影响使用，希望可以增加一个隐藏搜索栏关键词推荐的功能。多谢！辛苦了！

**Comment by Geeyun-JY3** [comment]
`隐藏热搜`组件就能视觉上隐藏原版顶栏搜索框的推荐：https://github.com/the1812/Bilibili-Evolved/commit/dd8822e375959849b80eabc8555a07b2bebef74f
